qemu/tests/balloon_hotplug.py

```diff
@@ -116,9 +116,10 @@ def enable_balloon_service():
         error_context.context("Unplug balloon device for %d times" % (i+1),
                               logging.info)
 
-        out = vm.devices.simple_unplug(devs[0].get_aid(), vm.monitor)
+        out = vm.devices.simple_unplug(devs[0].get_aid(), vm.monitor,
+                                       timeout=90)
```

Comment:
If RHEL linux guest take that long for unplug, should we be aware of it? If so I think better only give this long time out for windows.

Reply:
Updated, pls review, thanks.